mlflow/sklearn/utils.py

```diff
@@ -809,7 +809,7 @@ def first_custom_rank_column(df):
         excluded_metric_prefixes = ["param", "split"]
         metrics_to_log = {
             key: value
-            for key, value in result_row.iteritems()
+            for key, value in result_row.items()
```

Comment:
Older versions of pandas such as 1.3 also support this method:
https://pandas.pydata.org/pandas-docs/version/1.3/reference/api/pandas.Series.items.html

Reply:
Q: Does mlflow need to support pandas < 1.3 ?

Reply:
We currently don't have a policy on which versions to support. pandas <= 1.2 was released almost two years ago. I think it's fine not to support it.

Reply:
To say pandas < 1.3 is supported, we need to test it, but that would increase the maintenance burden.